### DISCUSSION TITLE
Changed email without having set up an email server, can't log back in

### DISCUSSION THREAD

**Discussion by b3nd1k** [firefly-iii/firefly-iii#5959] [Firefly III]
Hey!

I did the mistake of changing my log-in email in Firefly without having configured a mail server. Now I can't log back in because it says I haven't verified my email. How do I fhix this?

**Comment by niklas2810** [comment]
I'm not quite sure (and unfortunately can't check at the moment), but you should be able to update your email by hand in the database, if that's an option for you!
